test/api/clustering_test.cpp

```diff
@@ -14,6 +14,7 @@ int32_t const chrom1_position2 = 94734377;
 int32_t const chrom1_position3 = 112323345;
 std::string const chrom2 = "chr2";
 int32_t const chrom2_position1 = 234432;
+int16_t tandem_dup_count = 0;
```

Comment:
use `const`?
However, it would be good to write at least one test with more than 0 tandem duplications...